File: src/main/java/bad/robot/radiate/teamcity/BootstrapTeamCity.java
package bad.robot.radiate.teamcity;

@Deprecated
class BootstrapTeamCity extends TeamCity {

    public BootstrapTeamCity() {
        super(new BootstrapServer(), new EnvironmentVariableConfiguration().authorisation(), new HttpClientFactory().create(new EnvironmentVariableConfiguration()), new JsonProjectsUnmarshaller(), new JsonProjectUnmarshaller(), new JsonBuildUnmarshaller());
    }

}

File: src/main/scala/bad/robot/radiate/teamcity/BootstrapTeamCity.scala
package bad.robot.radiate.teamcity

class BootstrapTeamCityS extends TeamCityS(new BootstrapServerS, new EnvironmentVariableConfigurationS().authorisation, new HttpClientFactoryS().create(new EnvironmentVariableConfigurationS), new JsonProjectsUnmarshallerS, new JsonProjectUnmarshallerS, new JsonBuildUnmarshallerS)